landoapi/projects.py

```diff
@@ -32,35 +37,53 @@ def project_search(phabricator, project_phids):
     return result_list_to_phid_dict(phabricator.expect(projects, "data"))
 
 
-@cache.cached(key_prefix=SEC_PROJ_CACHE_KEY, timeout=SEC_PROJ_CACHE_TIMEOUT)
-def get_secure_project_phid(phabricator):
-    """Return a phid for the project indicating revision security.
+def get_project_phid(project_slug, phabricator, allow_empty_result=True):
+    """Looks up a project's PHID.
+
+    The result is cached so further lookups don't hit the network.
```

Comment:
I think you missed updating some things.

Reply:
I kept the refactoring isolated to this module to keep from complicating the larger PR.  If the refactoring is good I can apply it in other places as a follow-up PR.